ui/package.json

```diff
@@ -2,30 +2,28 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
-  "engines": {
```

Comment:
Restore the "engine" node. Please run "ui/tools/checkversion.js" to check the node version